Jamfile

```diff
@@ -210,10 +211,12 @@ rule linking ( properties * )
 	if <link>shared in $(properties)
 	{
 		result += <library>/try_signal//try_signal/<link>static/<fpic>on ;
+		result += <library>/libdatachannel//libdatachannel/<link>static/<fpic>on ;
```

Comment:
I'm not sure if this matters, but shouldn't this be conditioned on the `webtorrent` feature?